domains/bernardofoegbu.json

```diff
@@ -6,6 +6,8 @@
     "email": "obinnaofoegbu71@gmail.com"
   },
   "record": {
-    "CNAME": "o-bernardofoegbu.github.io"
+     "CNAME": "o-bernardofoegbu.github.io",
+     "CNAME_Label_Host": "bdvmkvxl5dlz",
+     "CNAME_Destination_Target": "gv-yrwqiastbtprak.dv.googlehosted.com"
```

Comment:
... these are not valid record types, 
please read our [documentation](https://is-a.dev/docs)